specification/gp-connect-access-record-fhir.yaml

```
@@ -1,13 +1,20 @@
 # This is an OpenAPI Specification (https://swagger.io/specification/)
 # for gp-connect-access-record-fhir owned by NHS Digital (https://digital.nhs.uk/)
 openapi: '3.0.0'
+x-spec-publication:
+  operation-order:
+    - operations:
+      - method: POST
+        path: /FHIR/STU3/Patient/$access-record-structured
```

This is the same operation as access record structured in GPC - if it's the same operation then the operation name needs to be consistent.

```suggestion
  /FHIR/STU3/Patient/$gpc.getstructuredrecord
```